docs/src/user-guide/guides-using-object-storage/compress.md

```diff
@@ -67,6 +69,7 @@ sbin/compress.sh s3 --aws-credentials-file <credentials-file> s3://<bucket-name>
     expose your credentials to other users running on the system).
 
 * `<bucket-name>` is the name of the S3 bucket containing your logs.
+* `<region-code>` is the AWS region [code][aws-region-codes] for the S3 bucket containing your logs.
 * `<key-prefix>` is the prefix of all logs you wish to compress.
```

Comment:
As discussed offline, the key-prefix here does not necessarily to be the same as the key-prefix specifies by the user.

One use case I imagine is that user has a bucket:/path/to/top_level_logs/* where they have different set of logs.
and maybe one invocation is that user wants to compress the logs of a specific application (maybe even within a specific timerange) by entering `sbin/compress bucket:/path/to/top_level_logs/my_application/log-2025-01-*`. Could be an over complicated use case though.